ArduPlane/quadplane.cpp

```diff
@@ -1292,3 +1300,37 @@ bool QuadPlane::verify_vtol_land(const AP_Mission::Mission_Command &cmd)
     check_land_complete();
     return false;
 }
+
+void QuadPlane::enable_geofence(void)
```

Comment:
actually, I like the separate function names. Although, for easier looking up may be better to start with "geofence" so that auto-lookup pulls up the rest easier. Also, please add a bool return with if it was successful since that may be in question on this verb.